### DISCUSSION TITLE
V2rayNG如何更新dat文件

### DISCUSSION THREAD

**Discussion by nudscn** [2dust/v2rayNG#3044] [Q&A]
Geo选项中，有个云下载的标志，但是下载从未成功过。云下载旁边的➕按钮，选“添加链接”，显示未找到文件，选“添加文件”，原来添加成功的一直都在，没有删除选项，添加新版文件，显示“备注已经存在”。

**Comment by 2dust** [comment]
选“添加文件”
从设备中选择了新的同名文件后会覆盖过去，这个文件不能删除，因为删除了core启动会失败

**Reply by nudscn** [reply]
如果更新太频繁，文件名列表会越来越多。能不能在文件的实际目录中，用新版文件覆盖旧版文件，以实现更新数据的目的？

**Comment by zazitufu** [comment]
首次挂全局代理，启动代理后，点击云朵是可以更新两个geo的dat文件的。
